projects/archiver/src/services/editions-mappings.ts

```diff
@@ -0,0 +1,37 @@
+const createEditionToNameMap = () => {
+    const map = new Map<string, string>()
+    map.set('daily-edition', 'Daily Edition')
+    map.set('american-edition', 'American Edition')
+    map.set('australian-edition', 'Australian Edition')
+    map.set('training-edition', 'Training Edition')
+    return map
+}
```

Comment:
This display name should definitely be in the template

Reply:
you mean as next step ?